### ISSUE TITLE
sundry documentation issues

### ISSUE THREAD

**Issue by tacaswell** [matplotlib/matplotlib#3531]
Noticed these when comparing pdf outputs from subsequent runs of building the docs.  Mostly put here so they don't get lost.
- some functions have function objects as default values which are leaking memory addresses into the docs.  These should probably be replaced by `None` and assigned the default value inside the function.
- the author list in the pdf docs needs a line break
- There are some examples that use un-seeded random number
- inheritance diagrams are missing (but that might just be the environment I am building in)

**Comment by rexissimus** [comment]
I would like to add some issues that are bothersome when trying to auto-parse the documentation:
* The parameter section for [boxplot](http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.boxplot) has wrong indentation and is not recognized as valid numpydoc. There are probably more issues like this.
* Some function signatures are missing default values for arguments, such as boxplot. Is this a regression, or should defaults no longer be specified in signatures?
* Defaults for numpydoc parameters are not always specified in the preferred numpydoc fashion, such as boxplot.whis, which make them hard to parse. Parameters in Matplotlib often supports many different value types, and it would be useful to have some rules on how these are specified in the documentation.

**Comment by anntzer** [comment]
@tacaswell Anything still relevant in the points you make, or can we close this issue?

**Comment by tacaswell** [comment]
maybe the pdf version, but nothing here is really actionable (and this is an _awful_ bug report!)
